airflow/hooks/__init__.py

```diff
@@ -27,6 +27,7 @@
 }
 
 _import_module_attrs(globals(), _hooks)
+from airflow.contrib.hooks import *
```

Comment:
Let's leave this line out until we retrofit `import_module_attrs` to play nicely with `contrib/`, or actually, people can explicitly reference `airflow.contrib.*` in their script, makes it more clear that it is community contributed.